src/lib/nexus-schema-backing-types/index.spec.ts

```diff
@@ -11,7 +11,7 @@ const ctx = TC.create(TC.tmpDir(), (ctx) => {
     setup(spec: FSSpec) {
       writeFSSpec(ctx.tmpDir, spec)
     },
-    async extractAndWrite(filePattern?: string) {
+    async extractAndWrite(filePattern: string = './**/*.ts') {
```

Comment:
Since these are unit tests I don't think we care this duplicates the default of Nexus. We can just consider it a coincidence/convenience. Fundamentally the unit tests just don't care.